README.md

```diff
@@ -13,6 +13,7 @@ Aka IFTTT orchestration for Ops.
 * Ubuntu Xenial (16.04)
 * RHEL6 / CentOS6
 * RHEL7 / CentOS7
+* RHEL8 / CentOS8
```

Comment:
Please try to also add CentOS8 builds in `.kitchen.yml` and `.travis.yml` so we'll get new OS support tested in this PR.